assets/css/main.css

```
@@ -12,195 +12,18 @@
 
 @config './tailwind.config.js';
 
-@theme {
-  --color-*: initial;
-  --color-transparent: transparent;
-  --color-current: currentColor;
-  --color-black: #000;
-  --color-white: #fff;
+@import './theme.css';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)